homeassistant/components/konnected/.translations/en.json

```diff
@@ -11,9 +11,13 @@
         },
         "step": {
             "confirm": {
-                "description": "Model: {model}\nHost: {host}\nPort: {port}\n\nYou can configure the IO and panel behavior in the Konnected Alarm Panel settings.",
+                "description": "Model: {model}\nID: {id}\nHost: {host}\nPort: {port}\n\nYou can configure the IO and panel behavior in the Konnected Alarm Panel settings.",
                 "title": "Konnected Device Ready"
             },
+            "import_confirm": {
+                "description": "A Konnected Alarm Panel with ID {id} has been discovered in configuration.yaml. This flow will allow you to import it into a config entry.",
+                "title": "Import Konnected Device"
+            },
```

Comment:
The import_confirm step was added to allow time for ssdp to discover host info for a panel.  This let's us utilize that host info when importing a configuration.yaml instead of forcing a user to enter it.